x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -445,4 +442,28 @@ export class FleetPlugin
     this.agentService = new AgentServiceImpl(internalEsClient);
     return this.agentService;
   }
+
+  private setupPackageService(
+    internalEsClient: ElasticsearchClient,
+    internalSoClient: any
```

Comment:
Why `any`? should be able to use a type given how its being called above.

Reply:
whoops, good catch 👍